### PR TITLE
MPE Adding Tracks - Temporal

### PR DESCRIPTION
**Pull request by Devessier**
Temporal part of #283

### CODE REVIEW

File: packages/temporal/api/mpe_routes.go [hunk 1]
```diff
@@ -100,6 +101,50 @@ func createMpeRoomHandler(w http.ResponseWriter, r *http.Request) {
...
+
+	signal := shared_mpe.NewAddTracksSignal(shared_mpe.NewAddTracksSignalArgs{
+		TracksIDs: body.TracksIDs,
+		UserID:    body.UserID,
+		DeviceID:  body.DeviceID,
+	})
+	if err := temporal.SignalWorkflow(
+		context.Background(),
+		body.WorkflowID,
+		"",
```
Comment (prastoin) [inline packages/temporal/api/mpe_routes.go:134]: should we create something like `shared.NoWorkflowRunIDRelated`

File: packages/temporal/mpe/workflows/mpe.go [hunk 2]
```diff
@@ -161,6 +168,84 @@ func MpeRoomWorkflow(ctx workflow.Context, params shared_mpe.MpeRoomParameters)
...
+
+					MpeRoomAddedTracksInformationFetchedEventType: brainy.Transition{
+						Actions: brainy.Actions{
+							brainy.ActionFn(
+								func(c brainy.Context, e brainy.Event) error {
+									event := e.(MpeRoomAddedTracksInformationFetchedEvent)
+
+									// If one track to add is already in the playlist, abort the whole operation.
+									// In practice, a single track can be added at a time.
+									// Therefore, aborting the whole operation makes sense.
```
Comment (prastoin) [inline packages/temporal/mpe/workflows/mpe.go:222]: I totally understand this point of view, this is completly ok.
I don't know if it's still the best behaviour. Even if we're not expecting any user to be able to send a tracks array, Temporal is agnostic about that. Maybe should we throw an error only if none of the fetched tracks could be added ?

File: packages/temporal/api/mpe_routes.go (hunk 1 above)
Comment (Devessier) [reply packages/temporal/api/mpe_routes.go:134]: That's a good idea, yeah!

File: packages/temporal/mpe/workflows/mpe.go (hunk 2 above)
Comment (Devessier) [reply packages/temporal/mpe/workflows/mpe.go:222]: It would be better, yeah!

**Review by prastoin — APPROVED** [review]
Great work ! very nice
Thank for your feedback
Ready to merge

**Pull request merged**